filemover/main.py

```diff
@@ -78,13 +77,15 @@ def get_folder(ext):
     return 'Other'
 
 
-def start():
+def start(folder_path: str): # need to change function name
```

Comment:
Nit/Pick: Adding TODO before the comment usually highlights these comments as tasks in IDEs